tutorials/assets_pipeline/retargeting_3d_skeleton.rst

```diff
@@ -0,0 +1,143 @@
+
+Unmapped Bones
+^^^^^^^^^^^^^^
+
+Removes unmapped bone transform tracks from the animations.
+
+Bone Renamer
+~~~~~~~~~~~~
+
+Rename Bones
+^^^^^^^^^^^^
+
+Rename the mapped bones.
+
+Unique Node
+^^^^^^^^^^^
+
+Makes Skeleton a unique node with the name specified in the Skeleton Name. This allows the animation track paths to be unified independent of the scene hierarchy.
+
+Rest Fixer
+~~~~~~~~~~
+
+Reference Poses defined in SkeletonProfileHumanoid have the following rules:
+
+* The humanoid is T-pose
+* The humanoid is facing +Z in the Right-Handed Y-UP Coordinate System
+* The humanoid should not have a transform as Node
+* Directs the +Y axis from the parent joint to the child joint
+* +X rotation bends the joint like a muscle contracting
+
+These rules are convenient definitions for blend animation and IK. If your model does not match this definition, you need to fix it with these options.
+
+Apply Node Transform
+^^^^^^^^^^^^^^^^^^^^
+
+If the asset is not exported correctly for sharing, the imported Skeleton may have a transform as a Node. For example, a glTF exported from Blender with no Apply Transform executed is one such case. It looks like the model matches the definition, but the internal transforms are different from the definition. This option fixes such models by applying transforms on importer.
+
+.. note:: If the imported scene contains objects other than Skeletons, this option may have a bad effect.
+
+Normalize Position Tracks
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Position track is used mostly for model movement, but sharing the moving animation between models with different heights may cause the appearance of slipping due to the difference in stride length. This option normalizes the Position track values based on the ``scale_base_bone`` height. The ``scale_base_bone`` height is stored in the Skeleton, and the normalized Position track values is multiplied by that value on playback. If this option is disabled, the Position tracks is not normalized and the Skeleton's ``motion_scale`` is always imported as ``1.0``.
+
+With SkeletonProfileHumanoid, ``scale_base_bone`` is Hips, therefore the Hips' height is used as the Motion Scale.
+
+Overwrite Axis
+^^^^^^^^^^^^^^
+
+Unifies the models' Bone Rest by overwriting it to match the Reference Pose defined in the SkeletonProfile.
+
+.. note:: This is the most important option for sharing animations in Godot 4.0+, but be aware that this option can produce horrible results **if the original Bone Rest set externally is important**. If you want to share animations with keeping the original Bone Rest, consider to use the `Realtime Retarget Module <https://github.com/TokageItLab/realtime_retarget>`__.
+
+Fix Silhouette
+^^^^^^^^^^^^^^
+
+Makes the model silhouette closer to the Reference Pose silhouette defined in the SkeletonProfile. This may not work well if the silhouettes are too different or for fixing roll.
```

Comment:
```suggestion
Attempts to make the model's silhouette match that of the reference pose defined in the SkeletonProfile, such as T-Pose. This cannot fix silhouettes which are too different, and it may not work for fixing bone roll.
```